app/_hub/kong-inc/openid-connect/_index.md

```diff
@@ -2058,9 +2061,9 @@ described in [the diagram](#authorization-code-flow) above.
 
 ### Password Grant
 
-Password grant is a legacy authentication grant. The password grant is a less
-secure way to authenticate the end users than the authorization code flow. For example,
-the passwords get shared with 3rd parties. The grant is rather simple though:
+Password grant is a legacy authentication grant. This is a less secure way of
+authenticating end users than the authorization code flow, because, for example,
+the passwords are shared with third parties. The grant is rather simple:
```

Comment:
```suggestion
Password grant is a legacy authentication grant. This is a less secure way of
authenticating end users than the authorization code flow, because, for example,
the passwords are shared with third parties. The image below illustrates the grant:
```